package.json

```diff
@@ -61,6 +61,7 @@
     "lodash": "^4.17.0",
     "ncp": "^2.0.0",
     "pre-commit": "^1.1.3",
-    "rimraf": "^2.5.4"
+    "rimraf": "^2.5.4",
+    "webpack": "2.1.0-beta.25"
```

Comment:
Installing with latest aegir still has the same problem as before: browser tests don't run. The error is
```
     Uncaught Error: Module '/Users/samuli/code/js-ipld-dag-pb/node_modules/buffer/index.js' is not a loader (must have normal or pitch function)
      at loadLoader (node_modules/loader-runner/lib/loadLoader.js:35:10)
      at iteratePitchingLoaders (node_modules/loader-runner/lib/LoaderRunner.js:164:2)
      at runLoaders (node_modules/loader-runner/lib/LoaderRunner.js:357:2)
      at NormalModule.doBuild (node_modules/webpack/lib/NormalModule.js:131:2)
      at NormalModule.build (node_modules/webpack/lib/NormalModule.js:179:15)
      at Compilation.buildModule (node_modules/webpack/lib/Compilation.js:127:9)
      at node_modules/webpack/lib/Compilation.js:306:10
      at node_modules/webpack/lib/NormalModuleFactory.js:74:13
      at NormalModuleFactory.applyPluginsAsyncWaterfall (node_modules/tapable/lib/Tapable.js:123:70)
      at onDoneResolving (node_modules/webpack/lib/NormalModuleFactory.js:49:11)
      at onDoneResolving (node_modules/webpack/lib/NormalModuleFactory.js:165:6)
      at node_modules/webpack/lib/NormalModuleFactory.js:161:6
      at node_modules/async/dist/async.js:3853:9
      at node_modules/async/dist/async.js:484:16
      at iteratorCallback (node_modules/async/dist/async.js:1084:13)
      at node_modules/async/dist/async.js:988:16
```